docs/advanced-node-scheduling.asciidoc

```diff
@@ -257,8 +257,8 @@ spec:
 
 This example relies on:
 
-- nodes from each zone being labeled accordingly. `failure-domain.beta.kubernetes.io/zone` link:https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#interlude-built-in-node-labels[is standard], but any label can be used.
-- node affinity for each group of nodes set to match the Kubernetes nodes zone.
+- Kubernetes nodes in each zone being labeled accordingly. `failure-domain.beta.kubernetes.io/zone` link:https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#interlude-built-in-node-labels[is standard], but any label can be used.
+- node affinity for each group of nodes set to match the Kubernetes node's zone.
```

Comment:
Is this referring to multiple k8s nodes? If yes, then I think it should be:
```suggestion
- node affinity for each group of nodes set to match the Kubernetes nodes' zone.
```

Reply:
It's unclear, it could be one k8s node in a zone or multiple. I think the plural works